### PR TITLE
keyboard_layout: add variants

### PR DESCRIPTION
**Pull request by lasers**
This is for @inigoserna's #1332.

### CODE REVIEW

**Comment by lasers** [comment]
@inigoserna I realize this PR is untested. This works okay for you?

**Comment by inigoserna** [comment]
Yes, it works perfectly. Thanks!

File: py3status/modules/keyboard_layout.py [hunk 1]
```diff
@@ -53,6 +53,7 @@
 
 import re
 LAYOUTS_RE = re.compile(r".*layout:\s*((\w+,?)+).*", flags=re.DOTALL)
+VARIANTS_RE = re.compile(r".*variant:\s*(([\w-]+,?)+).*", flags=re.DOTALL)
```
Comment (ultrabug) [inline py3status/modules/keyboard_layout.py]: bad kitty, please sort variables alphabetically

File: py3status/modules/keyboard_layout.py [hunk 2]
```diff
@@ -132,6 +133,11 @@ def _setxkbmap(self):
         layouts = re.match(LAYOUTS_RE, out).group(1).split(",")
         if len(layouts) == 1:
             return layouts[0]
```
Comment (ultrabug) [inline py3status/modules/keyboard_layout.py]: if you return here, how is that the rest of your commit works?

File: py3status/modules/keyboard_layout.py (hunk 1 above)
Comment (lasers) [reply py3status/modules/keyboard_layout.py]: Done.

File: py3status/modules/keyboard_layout.py (hunk 2 above)
Comment (lasers) [reply py3status/modules/keyboard_layout.py]: I hadn't touched this module so I don't know what the output should look like. I'll ask @inigoserna for `setxkbmap -query` output... and one more output again... with >1 layout if possible?

Comment (ultrabug) [reply py3status/modules/keyboard_layout.py]: Well I mean, your identation is the same so the return will always happen and your variant handling never be executed no?

Comment (inigoserna) [reply py3status/modules/keyboard_layout.py]: Sorry, can't understand.
The code I proposed is:
```
         if len(layouts) == 1:
-            return layouts[0]
+            variant = re.match(VARIANTS_RE, out)
+            if variant:
+                return layouts[0] + ' ' + variant.group(1)
+            else:
+                return layouts[0]
```

Comment (lasers) [reply py3status/modules/keyboard_layout.py]: @inigoserna I forget to remove `return layouts[0]` and @ultrabug caught that. Is fixed.

**Comment by ultrabug** [comment]
@inigoserna please confirm again with updated code before I merge?

**Comment by lasers** [comment]
@ultrabug Tested. Worked okay. Users can't specify colors though due to space and optionally unwanted dash. Is fixed.

**Comment by inigoserna** [comment]
Ok here.

**Comment by ultrabug** [comment]
:+1: thanks!

**Pull request merged**